providers/standard/src/airflow/providers/standard/operators/python.py

```diff
@@ -210,19 +220,18 @@ def execute(self, context: Context) -> Any:
     def determine_kwargs(self, context: Mapping[str, Any]) -> Mapping[str, Any]:
         return KeywordParameters.determine(self.python_callable, self.op_args, context).unpacking()
 
+    __prepare_execution: Callable[[], tuple[ExecutionCallableRunner, OutletEventAccessorsProtocol] | None]
```

Comment:
Any significance to `__` here vs `_`?

Reply:
No, just I want to weakly indicate this is for this class only, not subclasses (PythonOperator has many).